### ISSUE TITLE
import P5 * in fast_boid

### ISSUE THREAD

**Issue by MikeTheWatchGuy** [roholazandie/boids#2]
Would it be possible to refactor the fast version of your code too so that P5 is imported normally?

**Comment by roholazandie** [comment]
Yes, that's possible I will make the changes for the fast version just like the normal one.
